Automattic/wp-calypso
# Site profiler: Introduce globe block background

Closes https://github.com/Automattic/wp-calypso/issues/82514

## Proposed Changes

* Introduced globe hosting intro block background

## Testing Instructions



* Go to `/site-profiler`
* Hosting intro block: Check if there is a **globe background** instead of an image
* Check responsibility

<img width="1715" alt="Screenshot 2023-10-03 at 18 57 32" src="https://github.com/Automattic/wp-calypso/assets/1241413/b2cba3a9-b47f-4336-8c8d-659b0b0f105f">

<img width="509" alt="Screenshot 2023-10-03 at 18 57 24" src="https://github.com/Automattic/wp-calypso/assets/1241413/433fa3dd-21a6-436c-9c4f-56a662beef8e">


## Pre-merge Checklist
